app/Components/ModalComponent/ModalComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from './ModalComponent.module.scss';
+import Input from '../Input/Input';
+import { ModalComponentPropsInterface } from './interfaces/modal-component-props.interface';
+import { ModalComponentType } from './types/modal-component.type';
+import { InputTypeEnum } from '../Input/enum/input-type.enum';
+import Button from '../Button/Button';
+import { ButtonTypeEnum } from '../Button/enums/button-type.enum';
+import { isDarkState } from '@/app/States/States';
+import { useRecoilValue } from 'recoil';
+
+const ModalComponent: ModalComponentType = (
```

Comment:
also this modal is hard coded and it should not be hardcoded

Reply:
I don;t get it